bin/mkreleaselog

```diff
@@ -0,0 +1,233 @@
+#!/bin/zsh
```

Comment:
I'd put this in `scripts` to avoid creating more top-level directories

Reply:
Is anything in here zsh specific? We should target bash if not.